functests/sctp/sctp.go

```diff
@@ -116,88 +119,99 @@ var _ = Describe("sctp", func() {
 					return runningPod.Status.Phase
 				}, 1*time.Minute, 1*time.Second).Should(Equal(k8sv1.PodFailed))
 			})
-
-			AfterEach(func() {
-				namespaces.Clean("sctptest", client.Client)
-				deleteService(client.Client)
-			})
 		})
 	})
 	var _ = Describe("Test Connectivity", func() {
-		var activeService *k8sv1.Service
-		var serverPod *k8sv1.Pod
 		var nodes nodesInfo
 
 		execute.BeforeAll(func() {
 			checkForSctpReady(client.Client)
 			nodes = selectSctpNodes()
 		})
 
-		Context("Custom Namespace", func() {
+		Context("Connectivity between client and server", func() {
 			BeforeEach(func() {
-				activeService = createSctpService(client.Client)
-				By("Starting the server")
-				serverPod = startServerPod(nodes.serverNode, testNamespace)
+				namespaces.Clean(defaultNamespace, client.Client)
+				namespaces.Clean(testNamespace, client.Client)
 			})
 
 			// OCP-26759
 			It("Kernel Module is loaded", func() {
 				checkForSctpReady(client.Client)
 			})
+
 			// OCP-26760
-			It("Should connect a client pod to a server pod", func() {
-				testClientServerConnection(client.Client, testNamespace, serverPod.Status.PodIP,
-					activeService.Spec.Ports[0].Port, nodes.clientNode, serverPod.Name)
-			})
+			DescribeTable("connectivity test",
```

Comment:
@nikita133 can you please validate that this is according to our tests plan here